ArcGISRuntimeSDKQt_CppSamples/Features/FilterByDefinitionExpressionOrDisplayFilter/FilterByDefinitionExpressionOrDisplayFilter.cpp

```diff
@@ -133,3 +139,34 @@ void FilterByDefinitionExpressionOrDisplayFilter::queryFeatureCountInCurrentExte
 
   m_featureTable->queryFeatureCount(parameters);
 }
+
```

Comment:
This looks like it was supposed to be included in a different PR?

Reply:
Yes, branching off an active branch on accident. Should be cleared up now